treenode/admin.py

```diff
@@ -38,6 +49,15 @@ class MyModelAdmin(TreeNodeModelAdmin):
     list_per_page = 1000
     ordering = ('tn_order', )
 
+    def get_inlines(self, request, obj):
+        inlines = super().get_inlines(request, obj)
```

Comment:
This package supports python 2.7, so the super call must receive the 2 arguments.